Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.44858&attempt_id=ef9779dd-1dde-4eaa-b711-7c6186068526)

Attempted: sysdig

<details><summary>Partial log (click to expand)</summary><p>

```
                 from /build/source/userspace/libscap/scap_fds.c:19:
/nix/store/zxvpkal6ld7bhzivq9p332blxk64iwam-glibc-2.27-dev/include/bits/stdio2.h:64:10: note: '__builtin___snprintf_chk' output between 11 and 1102 bytes into a destination of size 1024
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [CMakeFiles/Makefile2:366: userspace/libscap/CMakeFiles/scap.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/syk3l6vxf2a0y373ibwbby2lkfp5r8cg-sysdig-0.22.1.drv' failed with exit code 2
error: build of '/nix/store/syk3l6vxf2a0y373ibwbby2lkfp5r8cg-sysdig-0.22.1.drv' failed
```
</p></details>

--- a/pkgs/os-specific/linux/sysdig/default.nix
+++ b/pkgs/os-specific/linux/sysdig/default.nix
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
     description = "A tracepoint-based system tracing tool for Linux (with clients for other OSes)";
     license = licenses.gpl2;
     maintainers = [maintainers.raskin];
-    platforms = platforms.linux ++ platforms.darwin;
+    platforms = ["x86_64-linux"] ++ platforms.darwin;
     downloadPage = "https://github.com/draios/sysdig/releases";
   };
 }

